ts/src/vector/featureTable.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { describe, it, expect } from "vitest";
+import { decodeTile } from "../index";
+import path from "node:path";
+import fs from "node:fs";
+
+const TILE = path.resolve(__dirname, "../../../test/expected/tag0x01/simple/multiline-boolean.mlt");
```

Comment:
This is basically tested in the mltDecoder code, isn't it? I think it would be better to focus here on unit tests, or if these are too complicated completely remove this test.
Generally speaking, I would like to have end to end tests like those done in the mltDecoder and the rest of the tests be unit tests that don't load files and don't perform network calls.
This is mainly from an ordering perspective.
If this test is important, move it to the mltDecoder.spec.ts file.